tequilapi/endpoints/node.go

```diff
@@ -78,6 +81,48 @@ func (ne *NodeEndpoint) MonitoringAgentStatuses(c *gin.Context) {
 	utils.WriteAsJSON(contract.MonitoringAgentResponse{Statuses: res}, c.Writer)
 }
 
+// GetProviderSessions A list of sessions metrics during a period of time
+// swagger:operation GET /node/provider/sessions GetProviderSessions
+// ---
+// summary: Provides Node sessions data during a period of time
+// description: Node sessions metrics during a period of time
+// parameters:
+//   - in: query
+//     name: range
+//     description: period of time ("1d", "7d", "30d")
+//     type: string
+// responses:
+//   200:
+//     description: Provider sessions list
+//     schema:
+//       "$ref": "#/definitions/ProviderSessionsResponse"
+//   400:
+//     description: Failed to parse or request validation failed
+//     schema:
+//       "$ref": "#/definitions/APIError"
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/APIError"
+func (ne *NodeEndpoint) GetProviderSessions(c *gin.Context) {
+	rangeTime := c.Param("range")
+
+	switch rangeTime {
+	case "1d", "7d", "30d":
```

Comment:
Would be nice to have such a thing as a constant somewhere, but this is ok I guess.